src/main/java/za/ac/cput/factory/VehicleFactory.java

```diff
@@ -0,0 +1,24 @@
+package za.ac.cput.factory;
+/*VehicleFactory.java
+  Factory class for the Vehicle
+  Author: Zintle Zothe (216130565)
+  Date: 07 April 2022
+ */
+import za.ac.cput.entity.Vehicle;
+import za.ac.cput.helper.GenerateID;
+
+
+public class VehicleFactory {
+    public static Vehicle createVehicle(String vehicleId, String vehiclePlateNumber, String vehicleType, String vehicleModel, String vehicleColour){
```

Comment:
Because the ID will be auto generated, maybe remove the String vehicleID parameter from the createVehicle() method.

Reply:
```suggestion
    public static Vehicle createVehicle(String vehiclePlateNumber, String vehicleType, String vehicleModel, String vehicleColour){
```